docs/users.md

```diff
@@ -0,0 +1,63 @@
+# Overview
+
+The following organizations are known to use Trino Gateway. The specific
+sections provide more details and references. 
+
+Some organizations may have stopped usage or requested not to be listed, while 
+many others are also using Trino Gateway without the Trino Gateway teams
+knowledge. 
+
+[Send a pull request or contact the Trino Gateway maintainers](development.md)
+for additions or changes to the user list.
+
+## BestSecret
+
+Isa is contributor, TCB epsiode and other talks?
+
+## Bloomberg
+
+Maintainer vishal, contributor andy, initial massive contribution, "founding 
+company" for the project
+
+## Expedia
+
+Contributor of UI changes
+
+## Line
```

Comment:
The name should be [LY](https://www.lycorp.co.jp/en/). LINE and Yahoo! JAPAN has been merged into one.